### ISSUE TITLE
Can't install extension with patched Safari

### ISSUE THREAD

**Issue by Gandalf99** [anakinsk/SafariStand#79]
I just tried to install an extension (Bear) in the patched version of Safari & it failed (see screen shot). When I tried installing it in the unpatched app, it worked. The good news (I assume)  is that when I exited and reloaded the patched version, the extension was there.

[SafariCan'tInstallBear.pdf](https://github.com/anakinsk/SafariStand/files/1886669/SafariCan.tInstallBear.pdf)

**Comment by anakinsk** [comment]
Right, currently I am working on a patch that will completely disable a library validation, which should allow us to use Safari without patching its launcher, which should fix this issue. Please stand by.

**Comment by kulkarni-sameer** [comment]
@Gandalf99 how do u switch back and forth between patched and unpatched versions of safari? I am also having the issues of not being able to install any extension on patched version.
